### PR TITLE
Fix check for primaries in subsets

### PR DESCRIPTION
**Pull request by sjledoux**
There is currently a bug in how primaries are checked for as members of sets, causing the exclusivity check to improperly pass submissions that violate our guidelines. This PR fixes the exclusivity check, and adds additional testing to verify it.

### CODE REVIEW

**Review by cfredric — APPROVED** [review]

**Review by Parasos — COMMENTED** [review]
related_website_sets.JSON

**Review by Parasos — COMMENTED** [review]
tests/rws_tests.py

File: tests/rws_tests.py [hunk 1]
```diff
@@ -330,6 +330,35 @@ def test_primary_is_associate(self):
         self.assertEqual(rws_check.error_list, 
          ["This primary is already registered in another"
                         + " related website set: https://primary2.com"])
+        
+    def test_primary_overlap(self):
```
Comment (Parasos) [inline tests/rws_tests.py:334]: @Parasos

**Review by Parasos — COMMENTED** [review]
tests/rws_tests.pyrelated_website_sets.JSONRwsCheck.py

**Pull request merged**